### ISSUE TITLE
Electron version uses wrong project generator command line executable

### ISSUE THREAD

**Issue by Meach** [openframeworks/projectGenerator#114]
Hi,

I was testing the new project generator done with Electron and found a bug.
I followed the readme to install it, I built an updated version of commandLine project beforehand, then moving the binary to frontend/app folder.

Then in the PG, when generating/updating a project, it uses frontend/app/projectGenerator...which gave me error such as: 

> 'projectGenerator' is not recognized as an internal or external command, operable program or batch file.

To fix this, I had to change in frontend/index.js (line 650):

> pgApp = pathTemp.normalize(pathTemp.join(pathTemp.join(__dirname, "app"), "projectGenerator"));

to this:
> pgApp = pathTemp.normalize(pathTemp.join(pathTemp.join(__dirname, "app"), "commandLine"));

So that it uses commandLine instead of projectGenerator. Then it was generating/updating projects correctly. 

I am on Windows 7 also, not sure if it matters.

**Comment by Meach** [comment]
Actually I just tried on OSX and it's all good: when compiling the commandLine project, it provides a projectGenerator binary, which then works correctly with the electron PG.

Then the problem is with Visual Studio for the commandLine project: the name of the executable should be "projectGenerator" and no "commandLine".

**Comment by ofZach** [comment]
can you open an issue here: https://github.com/openframeworks/projectGenerator ?  I think we do renaming this app as part of our packaging.

**Comment by arturoc** [comment]
yes, the bin name should stay as projectGenerator but for visual studio there's still no way to set the bin name when we generate the project file automatically
